src/components/scatterplot/ScatterplotOptions.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import Checkbox from '@material-ui/core/Checkbox';
+import Slider from '@material-ui/core/Slider';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableRow from '@material-ui/core/TableRow';
+
+const useStyles = makeStyles(theme => ({
+  box: {
+    boxSizing: 'border-box',
+  },
+  checkbox: {
+    padding: '3px',
+    color: theme.palette.primaryForeground,
+    '&:checked': {
+      color: theme.palette.primaryForeground,
+    },
+  },
+  slider: {
+    color: theme.palette.primaryForeground,
+    minWidth: '60px',
+    padding: '10px 0 6px 0',
+  },
+  sliderValueLabel: {
+    '& span': {
+      '& span': {
+        color: theme.palette.primaryBackground,
+      },
+    },
+  },
+  tableContainer: {
+    overflow: 'hidden',
+  },
+  labelCell: {
+    padding: '2px 8px 2px 16px',
+  },
+  inputCell: {
+    padding: '2px 16px 2px 8px',
+    overflow: 'visible',
+  },
+}));
+
+export default function ScatterplotOptions(props) {
+  const {
+    cellRadius,
+    setCellRadius,
+    cellSetLabelsVisible,
+    setCellSetLabelsVisible,
+    cellSetLabelSize,
+    setCellSetLabelSize,
+    cellSetPolygonsVisible,
+    setCellSetPolygonsVisible,
+  } = props;
+
+  const classes = useStyles();
+
+  function handleRadiusChange(event, value) {
+    setCellRadius(value);
+  }
+
+  function handleLabelVisibilityChange(event) {
+    setCellSetLabelsVisible(event.target.checked);
+  }
+
+  function handleLabelSizeChange(event, value) {
+    setCellSetLabelSize(value);
+  }
+
+  function handlePolygonVisibilityChange(event) {
+    setCellSetPolygonsVisible(event.target.checked);
+  }
+
+  return (
+    <Box className={classes.box}>
+      <TableContainer className={classes.tableContainer}>
+        <Table className={classes.table} size="small">
+          <TableBody>
+            <TableRow>
+              <TableCell className={classes.labelCell}>Cell Set Labels Visible</TableCell>
```

Comment:
Do we have an override for "Cell" somewhere?  I think we implemented perhaps some basic functionality for that but never went further.  Could we use that here?

Reply:
`observationsLabelOverride` I think?

Reply:
Good point. At some point we should do a more thorough check through the code base for this